src/crimson/osd/heartbeat.cc

```diff
@@ -229,7 +229,7 @@ seastar::future<> Heartbeat::ms_dispatch(ceph::net::ConnectionRef conn,
   logger().info("heartbeat: ms_dispatch {} from {}",
                 *m, m->get_source());
   switch (m->get_type()) {
-  case CEPH_MSG_PING:
+  case MSG_OSD_PING:
```

Comment:
```
src/include/ceph_fs.h:#define CEPH_MSG_PING                   2
...
src/msg/Message.h:#define MSG_OSD_PING         70
```

From `src/messages/MOSDPing.h`:
```cpp
  MOSDPing(const uuid_d& f, epoch_t e, __u8 o, utime_t s, uint32_t min_message)
    : MessageInstance(MSG_OSD_PING, HEAD_VERSION, COMPAT_VERSION),
      fsid(f), map_epoch(e), op(o), stamp(s), min_message_size(min_message)
  { }
  MOSDPing()
    : MessageInstance(MSG_OSD_PING, HEAD_VERSION, COMPAT_VERSION), min_message_size(0)
  {}
```

So it looks fine! I just regret we don't handle those codes as `enum class`.